web/src/main/java/se/inera/intyg/webcert/web/service/facade/user/UserStatisticsServiceImpl.java

```diff
@@ -71,12 +77,27 @@ public UserStatisticsDTO getUserStatistics() {
             return null;
         }
 
-        final var unitIds = getUnitIds(user);
+        final var careUnitIds = getCareUnitIds(user);
 
-        if (unitIds == null) {
+        if (careUnitIds.isEmpty()) {
             return null;
         }
 
+        final var useLimitedStatistics = careUnitIds.size() > maxCommissionsForStatistics;
```

Comment:
Should we update this variable to something that describes it better? Im thinking ```numberOfCommissionsExceeded```

Reply:
Or perhaps maxCommissionsExceeded